hack/tools/lifted-gen/lifted-gen.go

```diff
@@ -205,7 +207,7 @@ func (a *analyzer) parseComments(comments []*ast.CommentGroup) (item item, ok bo
 }
 
 func (a *analyzer) dump(path string) {
-	file, err := os.Create(path)
+	file, err := os.OpenFile(path, os.O_CREATE|os.O_RDWR, util.DefaultFilePerm)
```

Comment:
The `path` will contain the directory of the file, please help to confirm this:
- If the directory does not exist, will `os.OpenFile` also create the directory?
- How to control the permissions of the directory

Reply:
The logic here is consistent with the previous, confirmed no problem.

Reply:
thanks
/lgtm